source/addComment.js

```diff
@@ -1,3 +1,24 @@
 $(document).ready(function(){
-  // add your code here
+  $('#new_comment_button').on('click', function () {
+
+    var comment = '<textarea placeholder="Comment:" name="comment"></textarea><br>'
```

Comment:
storing these in variables work, but's it a little difficult to read on line 8. Feel free to put all that HTML in, and nest elements just as you would in an HTML document.